packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -73,11 +85,31 @@ export default class QuanticNumericFacet extends LightningElement {
         facetId: this.facetId ?? this.field,
       }
     });
+    if(!this.noInput) {
+      this.initializeFilter(engine);
+    }
+    this.searchStatus = CoveoHeadless.buildSearchStatus(engine);
     this.unsubscribe = this.facet.subscribe(() => this.updateState());
+    this.unsubscribeSearchStatus = this.searchStatus.subscribe(() => this.updateState());
+  }
+
+  /**
+   * @param {import("coveo").SearchEngine} engine
+   */
+  initializeFilter(engine) {
+     this.numericFilter = CoveoHeadless.buildNumericFilter(engine, {
+      options: {
+        field: this.field,
+        facetId: this.facetId ?? this.field
+      }
+    });
+    this.unsubscribeFilter = this.numericFilter.subscribe(() => this.updateState());
   }
 
   disconnectedCallback() {
     this.unsubscribe?.();
+    this.unsubscribeFilter?.();
+    this.unsubscribeSearchStatus?.();
   }
 
   updateState() {
```

Comment:
:thinking: Shouldn't you retrieve the filter state here (or use a different `updateState` function for the filter)?

After specifying a facet range, did you try reloading the page using the URL? Do the `start` and `end` values are reloaded correctly in the input fields?